src/Menu.php

```diff
@@ -42,6 +45,9 @@ class Menu implements Item, Countable, HasHtmlAttributes, HasParentAttributes
     /** @var bool */
     protected $activeClassOnLink = false;
 
+    /** @var bool */
```

Comment:
On second thought, I'd drop this part. Let's always add an exact active class. People can set an empty string if they don't want it.

Reply:
Ah, closure based activeness is handled somewhere else, found it. Now, I have no idea how to get my code from https://github.com/Joucke/menu/commit/6a814151f61364563942355ef0a6d98b08eb20d2 into this PR. Any help would be appreciated :)